crates/bevy_render/src/view/mod.rs

```diff
@@ -115,6 +116,26 @@ impl ExtractedView {
     }
 }
 
+#[derive(Component, Reflect, Debug, Copy, Clone, ShaderType)]
+#[reflect(Component)]
+pub struct ColorGrading {
+    pub exposure: f32,
+    pub gamma: f32,
+    pub pre_saturation: f32,
+    pub post_saturation: f32,
+}
```

Comment:
Is this all colour grading is? I was expecting histograms and stuff but I know nothing about this stuff. :)

Reply:
Suuuuper basic. We'll get fancy in 0.11